core-web/libs/sdk/client/src/lib/client/sdk-js-client.ts

```diff
@@ -198,6 +171,8 @@ export class DotCmsClient {
                     queryParamsObj['com.dotmarketing.persona.id'] = String(value);
                 } else if (key === 'mode' && value) {
                     queryParamsObj['mode'] = String(value);
+                } else if (key === 'variantName' && value) {
+                    queryParamsObj['variantName'] = String(value);
```

Comment:
We actually don't need this because is coming, we only do this if because the `personaId` needs to be changes to `'com.dotmarketing.persona.id'`, I don't think we need the `mode` either but we missed when @rjvelazco did it.

Reply:
I will remove then.